src/main/java/Duke/Storage.java

```diff
@@ -0,0 +1,69 @@
+package Duke;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Deals with loading and saving of tasks into a file.
+ */
+public class Storage {
+
+    /**
+     * Save data into a text file.
+     *
+     * @param tasks task list
+     */
+    public static void saveData(ArrayList<Task> tasks) {
+        try {
+            FileWriter fw = new FileWriter("savedData.txt");
+            StringBuilder sb = new StringBuilder();
+            for (int i=0; i<tasks.size(); i++) {
+                sb.append(tasks.get(i).saveDataFormat()).append(System.lineSeparator());
+            }
```

Comment:
It would be good to follow the format (including where whitespaces are used) for loops in the coding standard guide